mhyeun/twitter-emojify-api
# Add .env file and test file

### PR summary 📖 

This PR adds environment variables via the `python-dotenv` package.

An example for your `.env` file structure is provided in `.env.example`.

#### How to test this PR 🧪 
Make sure to install the new packages before testing, i.e. run:
```sh
pip install -r requirements.txt
```

Run `env_test.py`. It should output the `TWITTER_API_KEY` to your shell.
